pkg/permissions/src/accounts.ts

```diff
@@ -0,0 +1,35 @@
+import fetch, { Response } from 'node-fetch';
+import { Account } from './types';
+
+export const getAccountsWithPermissions = async (): Promise<Account[]> => {
+  const response: Response = await fetch('https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-authorizer', {
+    method: 'POST',
+
+    headers: {
+      'Content-Type': 'application/json',
+    },
+
+    body: JSON.stringify({
+      query: `{
+        accounts(first: 500) {
```

Comment:
What if 500 is not enough?

Reply:
We'd have to do pagination at some point as the graph limits how many responses is returns. We've got much fewer than 500 permissions atm (<50) so I'm not too concerned.

I'll add something to blow up this function if we actually receive 500 accounts.